Azure/sonic-buildimage
# [device]: Add LED initialzation file for Inventec-d7054q28b-r0

**- What I did**
* Add LED initialzation file for Inventec-d7054q28b-r0

**- How I did it**
* Update SAI profile (the configuration file is in this PR, https://github.com/Azure/sonic-buildimage/pull/1591)
* Add a new Initialization file for Broadcom LED microprocessors

**- How to verify it**
* Build image, install on device platform to and make sure it is configured properly

**- Description for the changelog**
* Add LED initialzation file for Inventec-d7054q28b-r0



**- A picture of a cute animal (not mandatory but encouraged)**
